docs/site/Decorators.md

```diff
@@ -340,74 +336,83 @@ Here's an example using 'BasicStrategy': to authenticate user in function
-its methods that communicate with your database. It is an interface to implement
-data persistence.
+LoopBack repository encapsulates your TypeScript/JavaScript Class instance,
+and its methods that communicate with your database.
+It is an interface to implement data persistence.
 
-Repository decorators are used for defining models(domain objects) for use with
+Repository decorators are used for defining models (domain objects) for use with
 your chosen datasources, and the navigation strategies among models.
 
 If you are not familiar with repository related concepts like `Model`, `Entity`
-and `Datasource`, please see LoopBack concept [Repositories](#Repositories.md)
+and `Datasource`, please see LoopBack concept [Repositories](Repositories.md)
 to learn more.
 
 ### Model Decorators
 
-Model is a Class that LoopBack builds for you to organize the data that share
-same configurations and properties. You can use model decorators to define a
-model and its properties.
+Model is a class that LoopBack builds for you to organize the data that
+share same configurations and properties. You can use model decorators to define
+a model and its properties.
 
 #### Model Decorator
 
 Syntax: `@model(definition?: ModelDefinitionSyntax)`
 
-Model decorator is a Class decorator. In LoopBack 4, we inherit the model
-definition format from LoopBack 3, you can find it in
+Model decorator is a class decorator. In LoopBack 4, we inherit the model
+definition format from LoopBack 3, which is described in the
 [Model definition JSON file](https://loopback.io/doc/en/lb3/Model-definition-JSON-file).
 For usage examples, see [Define Models](Repositories.md#define-models)
 
-_Please note we will elaborate more about model and model definition in
-\#Model.htm,_ _and replace the link above with LoopBack 4 link_
+*Please note we will elaborate more about model and model definition in the
+[Model](Model.md) page, and replace the link above with a LoopBack 4 link*
 
 By using a model decorator, you can define a model as your repository's
-metadata, then you have two choices to create the repository instance:
+metadata, which then allows you to choose between two ways of creating the
+repository instance:
 
-One is to inject your repository and resolve it with Legacy Juggler that
-complete with CRUD operations for accessing the model's data. A use case can be
-found in section [Repository decorator](#repository-decorator)
+One is to inject your repository and resolve it with Legacy Juggler that's
+complete with CRUD operations for accessing the model's data.
+A use case can be found in section [Repository decorator](#repository-decorator)
 
-The other one is defining your own repository without using legacy juggler, and
-use an ORM/ODM of your choice.
+The other one is defining your own repository without using legacy juggler,
+and use an ORM/ODM of your choice.
 
 ```ts
 // Missing example here
```

Comment:
It feels weird that this comment is split into two lines. feel free to ignore though.